widget/edit.html

```diff
@@ -38,35 +38,41 @@ <h3 class="modal-title col-md-9">Edit Funnel Chart</h3> <button type="button" cl
                 </select>
             </div>
             <div class="form-group" data-ng-if="config.customModule !== undefined">
-                <div class="margin-top-md margin-bottom-md">
-                    <label class="control-label">Filter Record Which Contains The Data
-                        <span class="text-danger">*</span>
-                    </label>
-                    <div data-cs-conditional data-ng-if="fieldsArray.length > 0" data-fields="$parent.fields"
-                        data-reset-field="$parent.fields" data-mode="'queryFilters'"
-                        data-ng-model="$parent.config.query" data-parent-form="editFunnelWidgetForm"
-                        data-enable-expression="(page==='dashboard' || page==='reporting')" data-show-uuid="true"
-                        data-form-name="'editFunnelWidgetForm'">
+                <div class="form-group" data-ng-if="objectFields.length > 0">
+                    <div class="margin-top-md margin-bottom-md">
+                        <label class="control-label">Filter Record Which Contains The Data
```

Comment:
what about "Filter record which contains the JSON data"?

Reply:
Done